cwl/src/main/scala/wdl4s/cwl/FullyQualifiedName.scala

```diff
@@ -0,0 +1,81 @@
+package wdl4s.cwl
```

Comment:
ToL: Can we get some doc about what the logic in those methods / classes is about ?
Just a few one liners is enough but to be able to make sense of all the `split`ting going on :)